### ISSUE TITLE
training base network (ssd mobilenet v1)

### ISSUE THREAD

**Issue by seongkyun** [qfgaohao/pytorch-ssd#11]
How can i get mobilenet base network's weights by myself?

Is that just result of training with PASCAL VOC 0712 data set classification task?

I've tried ssd mobilenet v1 end-to-end without your base network weight file and it's result was mAP 0.57.
(options: --batch_size 32 --num_epochs 400 --scheduler cosine --lr 0.001 --t_max 200)
But with your pretrained base network weights(mobilenet-v1-ssd-mp-0_675.pth), the mAP was 0.67 that you wrote on main page samely.
(options: --batch_size 32 --num_epochs 200 --scheduler cosine --lr 0.01 --t_max 200)

And can I get or download trained VGG16 based SSD network's weight file?

Best regards,

**Comment by qfgaohao** [comment]
Hi @seongkyun you can download the pertained VGG16 model here 
https://s3.amazonaws.com/amdegroot-models/vgg16_reducedfc.pth . This is the one I am using.

Weights initialisation is an open question in NN. There are many research works around it. Generally, it's a good idea to use a pertained model to initialise the weights or part of the weights and make a reasonable start in the early stage of training.

**Comment by qfgaohao** [comment]
@seongkyun plz see the trained model with the state of the art MP.
https://storage.googleapis.com/models-hao/vgg16-ssd-mp-0_7726.pth

Here is script to train it, just in case you want to train it on your own.
python train_ssd.py --datasets ~/data/VOC0712/VOC2007/ ~/data/VOC0712/VOC2012/ --validation_dataset ~/data/VOC0712/test/VOC2007/ --net vgg16-ssd --base_net models/vgg16_reducedfc.pth  --batch_size 24 --num_epochs 200 —scheduler "multi-step” —-milestones “120,160”

**Comment by seongkyun** [comment]
@qfgaohao  Thank you!
